app/monitoring/CloudWatch.scala

```diff
@@ -0,0 +1,65 @@
+package monitoring
+
+import java.util.concurrent.Future
+
+import com.amazonaws.handlers.AsyncHandler
+import com.amazonaws.regions.Regions.EU_WEST_1
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchAsyncClient
+import com.amazonaws.services.cloudwatch.model.{Dimension, MetricDatum, PutMetricDataRequest, PutMetricDataResult}
+import com.gu.aws.CredentialsProvider
+import monitoring.CloudWatch.cloudwatch
+
+trait CloudWatch extends TagAwareLogger {
+  val stage: String
+  val application : String
+  lazy val stageDimension: Dimension = new Dimension().withName("Stage").withValue(stage)
+  def mandatoryDimensions:Seq[Dimension] = Seq(stageDimension)
+
+  trait LoggingAsyncHandler extends AsyncHandler[PutMetricDataRequest, PutMetricDataResult]
+  {
+    def onError(exception: Exception)
+    {
+      logger.info(s"CloudWatch PutMetricDataRequest error: ${exception.getMessage}}")
+    }
+    def onSuccess(request: PutMetricDataRequest, result: PutMetricDataResult )
+    {
+      logger.trace("CloudWatch PutMetricDataRequest - success")
+      CloudWatchHealth.hasPushedMetricSuccessfully = true
+    }
+  }
+
+  object LoggingAsyncHandler extends LoggingAsyncHandler
+
+
+  def put(name : String, count: Double, extraDimensions: Dimension*): Future[PutMetricDataResult] = {
```

Comment:
Could the full class path for`Future` be used or it be imported as `JFuture`? otherwise I think someone will just assume it's a Scala future